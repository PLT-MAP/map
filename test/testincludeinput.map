include header.map

func main(){
	Text line=test();
	print(line);
	}






