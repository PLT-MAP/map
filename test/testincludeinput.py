import networkx as nx
import sys
def test():
    string = raw_input("Enter a string")
    return string
def main():
    test()
if __name__ == '__main__': 
	try:
		main()
	except:
		print'Error:',sys.exc_info()[0]
		print'Resolve error before running again!'