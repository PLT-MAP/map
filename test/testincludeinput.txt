def test():
	string = raw_input("Enter a string: ")
	return string
def main():
	line = test()
	print(line)
