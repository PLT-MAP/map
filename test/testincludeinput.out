import networkx as nx
import sys
import matplotlib.pyplot as plt
def test():
    string = raw_input("Enter a string: ")
    return string
def main():
    line = test()
    print(line)
if __name__ == '__main__': 
	try:
		main()
	except:
		print'Error:',sys.exc_info()[0]
		print'Resolve error before running again!'